std/file.d

```diff
@@ -5079,7 +5080,7 @@ auto dirEntries(bool useDIP1000 = dip1000Enabled)
     foreach (string name; dirEntries(testdir, SpanMode.breadth))
     {
         //writeln(name);
-        assert(name.startsWith(testdir));
+        assert(name.startsWith(absolutePath(testdir)));
```

Comment:
This needs to be fixed. As @CyberShadow pointed out, we aren’t supposed to break existing code here: <https://github.com/dlang/phobos/pull/6125#pullrequestreview-93860905>